dev/sample/launch.json

```diff
@@ -0,0 +1,18 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "DEBUG SDP-API",
+            "type": "go",
+            "request": "attach",
+            "mode": "remote",
+            "remotePath": "/app/github.com/stellar/stellar-disbursement-platform",
+            "localRoot": "/${workspaceFolder}/dev/sdp-debugging-pr/stellar-disbursement-platform-backend", 
```

Comment:
Is this the path where the project is cloned in the developer's machine ? If so, can we add a comment to ask people to cutomize this value.

Reply:
let me look into this. it should be generic.

Reply:
thanks for catching it